src/plugins/installer.py

```diff
@@ -0,0 +1,92 @@
+import os
+import pathlib
```

Comment:
Is there a reason you avoid import-from statements? In my opinion they help to reduce clutter in the code
```suggestion
from pathlib import Path
```